app/routes/polls/components/PollsList.tsx

```diff
@@ -5,100 +5,103 @@ import {
   Icon,
   LoadingIndicator,
 } from '@webkom/lego-bricks';
+import { useEffect } from 'react';
 import { Helmet } from 'react-helmet-async';
 import { Link } from 'react-router-dom';
+import { fetchAll } from 'app/actions/PollActions';
 import { Content } from 'app/components/Content';
 import NavigationTab from 'app/components/NavigationTab';
 import Paginator from 'app/components/Paginator';
-import type { ActionGrant } from 'app/models';
-import type { PollEntity } from 'app/reducers/polls';
+import { selectPolls } from 'app/reducers/polls';
+import { useAppDispatch, useAppSelector } from 'app/store/hooks';
 import styles from './PollsList.css';
 
-type Props = {
-  polls: Array<PollEntity>;
-  actionGrant: ActionGrant;
-  fetching: boolean;
-  hasMore: boolean;
-  fetchAll: (arg0: { next?: boolean }) => Promise<any>;
-};
+const PollsList = () => {
+  const polls = useAppSelector((state) => selectPolls(state));
```

Comment:
```suggestion
  const polls = useAppSelector(selectPolls);
```

Reply:
ah I didn't know this syntax was accepted - good to know!